src/merge.py

```
@@ -544,41 +581,15 @@ def main():
-                else:
-                    icloud_event.action = EventAction.delete
-                merge_events.append(icloud_event)
-
-            events_to_add = [event for event in source_calendar_events if event.action is None]
-            event_addition = len(events_to_add) > 0
-        else:
-            event_addition = True
-            events_to_add = source_calendar_events
         term.print_done()
 
+        print_step(TAG_CALENDAR_MERGE, f"reconciling events...", False)
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)